docs/sources/tempo/traceql/_index.md

```diff
@@ -119,6 +119,25 @@ For example, to find traces with an attribute of `sla` set to `critical`:
 { .sla = "critical" }
 ```
 
+### Quoted attribute names
+
+It's possible that attributes name can contain terminal characters, to search span attributes with terminal character you can use quoted attribute syntax. A quoted attirbute should be enclosed inside double quote and all characters between the quotes will be considered part of the attribute name.
+
+For example, to find span with attribute name `attribute name with space` one can use the following query
```

Comment:
```suggestion
Attribute names can contain terminal characters. To search span attributes with terminal characters, you can use quoted attribute syntax. A quoted attribute should be enclosed inside double quotes, for example, `"example one"`. All characters between the quotes are considered part of the attribute name.

For example, to find span with attribute name `attribute name with space`, you can use the following query:
```

Reply:
Thanks a lot. 🙂 I have made the changes and will refer to the mentioned style guide going forward.